package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "moment": "^2.29.1"
+  }
+}
```

Comment:
get rid of this (in the top level folder)

Reply:
Remove the package-lock.json too